README.md

```diff
@@ -170,7 +170,7 @@ scene.add( tilesRenderer2.group );
 
 ## Adding DRACO Decompression Support
 
-Adding support for DRACO decompression within the GLTF files that are transported in B3DM and I3DM formats. The same approach can be used to add support for KTX2 and DDS textures.
+Adding support for DRACO decompression within the GLTF files that are transported in B3DM and I3DM formats. The same approach can be used to add support for KTX2 and DDS textures. Note that GLTF decoding and PNTS decoding require separate decoders.
```

Comment:
I'm not sure what this added line means. Can it be removed or clarified?

Reply:
In the current docs, the selected draco decoder from unpkg uses the GLTF branch, which *does not work* for compressed PNTS files. This is a really easy detail to miss IMO, I added this line to clarify that. I'll reword things

https://github.com/mrdoob/three.js/tree/dev/examples/jsm/libs/draco
vs
https://github.com/mrdoob/three.js/tree/dev/examples/jsm/libs/draco/gltf

Reply:
Oh I see now... how annoying. I'll move this information down to the section below where the PNTS draco loader is being discussed.